src/components/APIViews/Search/SearchDisplay/Results/index.js

```diff
@@ -2,24 +2,26 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import typy from 'typy'
+import ResponsiveGridList from 'components/Shared/ResponsiveGridList'
 import Result from './Result'
 
 export const Results = ({ searchReducer, location }) => {
   if (typy(searchReducer, 'results.docs').safeObject) {
     return (
-      <React.Fragment>
+      <ResponsiveGridList>
         {
           searchReducer.results.docs.map((doc, index) => {
             return (
-              <Result
-                key={index}
-                doc={doc}
-                location={location}
-              />
+              <div key={`${index}`}>
```

Comment:
I think this could be susceptible to issues if the results get sorted, filtered, etc. Is there an id on the doc or something that could be used as a key instead of index?

Reply:
Sorting and filtering problems with keys aren't currently relevant since new results would be generated through an api request triggering a higher level re-render.

I don't like using `index` here, but in order to build the `layouts` object required by `react-grid-layout` I need consistent keys for the `ResponsiveGridList` component. Passing an array of ids is an option, but it adds an additional step to use the component as that array would have to be built separately from the mapping of child node.